packages/camera/camera_web/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: camera_web
+description: A Flutter plugin for getting information about and controlling the camera on Web.
+repository: https://github.com/flutter/plugins/tree/master/packages/camera/camera_web
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
+version: 0.0.1
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.0.0"
+
+flutter:
+  plugin:
+    platforms:
+      web:
+        pluginClass: CameraPlugin
+        fileName: camera_web.dart
+
+dependencies:
+  camera_platform_interface: ^2.0.1
+  flutter:
+    sdk: flutter
+  flutter_web_plugins:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  mocktail: ^0.1.4
```

Comment:
No need for mocktail in the direct dependencies of the package, only in the `example`.